ansible/group_vars/all

```diff
@@ -24,18 +26,25 @@ nginx_vhosts:
     return: "302 http://$hostname"
   - listen: "80"
     server_name: "$hostname"
-    index: "index.php index.htm index.html"
-    root: "/var/www/biblebox"
-    error_page: "404 /biblebox_default/content/index.html"
+    index: index.html
+    root: "{{ biblebox_default_content_root }}"
+    error_page: "404 /index.html"
     access_log: "/var/log/nginx/biblebox-access.log"
     error_log: "/var/log/nginx/biblebox-error.log"
     extra_parameters: |
       rewrite_log on;
 
-      # pass the PHP scripts to FastCGI server listening on 127.0.0.1:9000
+      # Admin interface
+      location /admin/ {
+        root {{ biblebox_web_root }};
```

Comment:
When you use `root`, the path on disk is the value of the `root` directive and the URL. The url starts with /admin/ in this case, so if we used `{{ biblebox_admin_root }}`, nginx would be looking for files under `/var/www/biblebox/admin/admin`. The `alias` directive doesn't append the URL when nginx is looking for files on disk so `alias {{ biblebox_admin_root }}` is equivalent to what we have. Perhaps you're thinking of `alias`?